product-releases/2021-04-08-sonarqube-support.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Support for Sonarqube
+author: William Li
+author_title: Product Engineering Lead
+author_url: https://github.com/weeyum
+author_image_url: https://assets-global.website-files.com/5fd1d1ab54430e6b747263b2/5fd1d1ab54430ee03472641b_william%20headshot.png
+tags: []
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In the past few days, the effx platform engineering team landed support for [Sonarqube][].
+Sonarqube is an open-source platform developed by SonarSource for continuous inspection of code quality.
+Sonar does static code analysis, which provides a detailed report of bugs, code smells, vulnerabilities, code duplications.
```

Comment:
`Sonar performs static code analysis`

Reply:
I'd also add a line after this about how effx uses this information and turns it into more value for the customer. I was having trouble coming up with a line for this which was why I started looking at adding tags.

Reply:
same, the current approach doesn't add too much yet. perhaps we can come back and update the value prop once we have a better selling point? 

I think ideally it'd work similar to SLOs with badges